templates/layouts/page.html

```diff
@@ -37,6 +37,24 @@
     <noscript id="ns-main">
         <span>Code Golf works best with JavaScript enabled.</span>
     </noscript>
+    <!--[if lte IE 9]>
+    <div id="upgrade">
+        <div class="upgrade-wrapper">
+            <div><h1>Your browser is not supported</h1></div>
+            <div><h3>The following browsers are officially supported:</h3></div>
+            <div class="browser-list">
+                <div class="col-ff">Firefox<span>&gt; 27</span></div>
```

Comment:
Rather than listing browsers that aren't supported, we should list browsers that are (using `>=` instead of `>`).

Reply:
`if browser < minimum version: display message` is the same as `if not (browser >= minimum version): display message`

Reply:
The issue is that the warning says `browser > version` instead of `browser >= version`.